contrib/pkcs11-helper/template.py

```diff
@@ -0,0 +1,32 @@
+pkgname = "pkcs11-helper"
+pkgver = "1.30.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+        "--disable-static",
+        "--enable-threading",
+        "--enable-slotevent",
+]
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = ["automake", "pkgconf", "libtool", "gmake"]
+makedepends = ["openssl-devel"]
+pkgdesc = "Helper library for multiple PKCS#11 providers"
+maintainer = "Dmitriy Vakhrushev <dvakhrushev@netgate.com>"
+license = "GPL-2.0-only AND BSD-3-Clause"
+url = "https://github.com/OpenSC/pkcs11-helper"
+source = f"{url}/archive/refs/tags/pkcs11-helper-{pkgver}.tar.gz"
+sha256 = "076c9f664812a45f2da25efc157338b0b8bb1949117f0144050fec176b6fdf78"
+
```

Comment:
check if hardening= works

Reply:
it works at least locally, ks, but I've changed

Reply:
it's not related to the url (keep the `{url}`). see e.g. `contrib/ddrescue`. for `vis`, see if all the exported symbols are there (`nm -D` on the library before/after), for cfi, see if it works without crashing (usually a testsuite would catch that if one runs here)

Reply:
got it, thank you very match for the details.

Reply:
flags have been checked:

-  tests can be enabled only when some kind of pkcs11 provider is available on the system
   I've tested on that it works on ubuntu system with preinstalled OpenHSM module. I've tried to add this module in the package, but it isn't trivial and needed to spend more time, which I have no for now.

- so I've tested it just by seeing at least that the tests are started and dynamic linkage passed. These  hardening (vis,cif) makes tests failed on dynamic linking, because part of the necessary functions are hidden. I guess this is what you might have guessed.

- 'vis' is closing major part of functions

- adding 'cif' is closing another several functions

Reply:
makes sense, thanks :)